test/pgo_trace_SUITE.erl

```diff
@@ -13,6 +13,7 @@ all() ->
 
 init_per_suite(Config) ->
     application:ensure_all_started(pgo),
+    application:ensure_all_started(tls_certificate_check),
```

Comment:
https://github.com/erleans/pgo/pull/85#issuecomment-2313932296 <- handles this error